src/API/Reports/Coupons/Stats/DataStore.php

```diff
@@ -177,12 +177,24 @@ public function get_data( $query_args ) {
 				return $data;
 			}
 
```

Comment:
Can you include the same `@todo` comment here as in https://github.com/woocommerce/woocommerce-admin/pull/2961/commits/2d5819a9eca01846b4fa33164adacc8971fe2dd2#diff-095b9033af163cc91be665761af1bb8eR163 ? It took a few minutes to figure out why this was needed.

Reply:
Added in https://github.com/woocommerce/woocommerce-admin/pull/2961/commits/e1a4d9aac2622acbdd03259b07ce25c9470547ee